dev/com.ibm.ws.security_fat.common.tooling/bnd.bnd

```diff
@@ -0,0 +1,50 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src:\
+  src,\
+  test-applications/testmarker.war/src
+
+
+test.project: true
+
+
+#   lib/jdom-1.1.2.jar;version=file,\
+-buildpath: \
+  com.ibm.ws.com.meterware.httpunit.1.7;version=latest,\
+  com.ibm.ws.logging;version=latest,\
+  com.ibm.ws.mongo_fat;version=latest,\
+  com.ibm.ws.org.joda.time.1.6.2;version=latest,\
+  com.ibm.websphere.javaee.servlet.3.0;version=latest,\
+  com.ibm.websphere.org.osgi.core;version=latest,\
+  com.ibm.websphere.org.osgi.service.component;version=latest,\
+  com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
+  commons-lang:commons-lang;version=2.4,\
+  fattest.simplicity;version=latest,\
+  org.jdom:jdom;version=1.1.2,\
+  lib/htmlunit-2.20-OSGi.jar;version=file
```

Comment:
Why using a binary here instead of depending on a maven / dhe repository location?

Reply:
They made some changes to this JAR, I am not entirely certain what those changes were. I pulled in the one they were using over in WS.

Reply:
You can still publish a version of it into the dhe / artifactory repo.

Reply:
I removed this project since parts of it had been created already.